azurerm/resource_arm_storage_account.go

```diff
@@ -1406,13 +1406,14 @@ func expandQueuePropertiesCors(input []interface{}) *queues.Cors {
 }
 
 func flattenStorageAccountNetworkRules(input *storage.NetworkRuleSet) []interface{} {
-	if len(*input.IPRules) == 0 && len(*input.VirtualNetworkRules) == 0 {
-		return []interface{}{}
-	}
```

Comment:
🤔 reading this again I believe this wants to check if `input` is nil, rather than that these specific properties are nil